lib/users.rb

```diff
@@ -0,0 +1,26 @@
+require 'rubygems'
+require "data_mapper"
+
+if ENV['ENVIRONMENT'] == 'test'
```

Comment:
As per comment below (for some reason I am reading upwards), this is duplicate code from your chitter.rb, so probably belongs in a separate connection class.